plugins/eventful.cpp

```diff
@@ -112,32 +112,32 @@ DEFINE_LUA_EVENT_NH_0(onUnload);
 DEFINE_LUA_EVENT_NH_6(onInteraction, std::string, std::string, int32_t, int32_t, int32_t, int32_t);
 
 DFHACK_PLUGIN_LUA_EVENTS {
-    DFHACK_LUA_EVENT(onWorkshopFillSidebarMenu),
```

Comment:
Please undo unnecessary formatting changes. It's hiding the important parts of the diff